goal_src/jak1/engine/math/math.gc

```diff
@@ -317,3 +317,68 @@
   `(+ -1.0 (the-as float (logior #x3f800000 (/ (rand-uint31-gen ,gen) 256))))
   )
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;; mimicking Java's util.Random
+;;;; https://docs.oracle.com/javase/8/docs/api/java/util/Random.html
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+(define *java-rand-seed* (the uint64 42069))
```

Comment:
do we have a system time or anything like that accessible, that we could use here as a more dynamic default seed?

Reply:
worst case: `0x666edd1e`? 😉

Reply:
Hmm I think math.gc is handled before all_types.gc, getting an error `Type Error: Type scf-time is not fully defined.` if I try to use `scf-time` within math.gc 😞

Reply:
Yeah, `math.gc` is one of the earliest files that gets compiled and `scf-time` is defined in `settings-h.gc`.
You'd have to move the type definition of `scf-time`.